Reflection_Engine/Modify/SetPropertyValue.cs

```diff
@@ -86,6 +86,25 @@ public static object SetPropertyValue(this object obj, string propName, object v
                         value = Activator.CreateInstance(propType);
                 }
 
+                if (propType.IsEnum && value is string)
+                {
+                    string fullName = value as string;
+                    int index = fullName.LastIndexOf('.');
+                    if (index > 0)
+                    {
+                        Type enumType = Create.Type(fullName.Substring(0, index));
+                        if (enumType != null)
+                        {
+                            object enumValue = Enum.Parse(enumType, fullName.Substring(index + 1));
+                            if (enumValue != null)
+                                value = enumValue;
+                        }
+                    }
```

Comment:
Any reason to not just use the property type, instead of trying to create a type from the providede value?

Have not tested this, just looked on github, but should nto something like this work?

```suggestion
                    string enumName= (value as string).Split('.').Last();
                    object enumValue = Enum.Parse(propType, enumName);
                    if (enumValue != null)
                        value = enumValue;
```